packages/gafl-webapp-service/src/pages/contact/address/entry/route.js

```diff
@@ -18,11 +18,16 @@ const validator = Joi.object({
   'country-code': Joi.string().required()
 }).options({ abortEarly: false, allowUnknown: true })
 
+const getCountryDropDownOptions = async () => {
+  const options = await countries.getAll();
+  return options.filter(country => country.code !== 'GB')
+}
+
 export default pageRoute(ADDRESS_ENTRY.page, ADDRESS_ENTRY.uri, validator, nextPage, async request => {
   const { addresses, searchTerms } = await request.cache().helpers.addressLookup.getCurrentPermission()
   return {
     searchTerms: !addresses?.length && searchTerms ? searchTerms : null,
-    countries: await countries.getAll(),
+    countries: await getCountryDropDownOptions(),
```

Comment:
Would be good to have a new test for this

Reply:
Ah, yes, good catch. I think I got carried away with finishing this ticket. I'll add that in :)